server/routes/courtRegister/amendCourtDetailsValidator.ts

```diff
@@ -0,0 +1,31 @@
+import { Request } from 'express'
+import type { AmendCourtDetailsForm } from 'forms'
+
+export default async function validate(
+  form: AmendCourtDetailsForm,
+  req: Request,
+  updateService: (id: string, name: string, type: string, description: string) => Promise<void>
+): Promise<string> {
+  const errors: Array<{ text: string; href: string }> = []
+
+  if (form.name.trim().length === 0) {
```

Comment:
I should be able to extract these since they are the same when we create a court